sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -213,7 +213,14 @@ def update(self, other):
       DeferredSeries,
       '__i%s__' % base,
       frame_base._elementwise_method('__i%s__' % base, inplace=True))
-for name in ['__lt__', '__le__', '__gt__', '__ge__', '__eq__', '__ne__']:
+for name in ['__lt__',
+             '__le__',
+             '__gt__',
+             '__ge__',
+             '__eq__',
+             '__ne__',
+             '__or__',
```

Comment:
Add `__ior__` and `__iand__` as well?

Reply:
Moved `or` and `and` so that `__ior__` and `__iand__` get defined